.github/workflows/test.yml

```diff
@@ -45,6 +45,19 @@ jobs:
           - "tox"
           - "min"
 
+    continue-on-error: >- # jobs not required in branch protection
+      ${{
+        (
+          (
+            startsWith(matrix.py, 'pypy-')
+            && (!endsWith(matrix.py, '-3.7') || matrix.os == 'windows')
+          )
+          || matrix.py == '3.11'
```

Comment:
You don't have to exclude 3.11 here - we just forgot to add it to the protection rules.

Reply:
Ah, I'll drop the line, then. Hopefully, once this is merged, it'll be harder to forget such things since they won't be hidden somewhere in the repo settings only a few folks get to see.

Reply:
Done!